config/config.go

```diff
@@ -397,6 +398,8 @@ type GlobalConfig struct {
 	ScrapeProtocols []ScrapeProtocol `yaml:"scrape_protocols,omitempty"`
 	// How frequently to evaluate rules by default.
 	EvaluationInterval model.Duration `yaml:"evaluation_interval,omitempty"`
+	// Initial delay before starting the evaluation of rules to ensure the underlying metrics have been received.
```

Comment:
The misleading old "Initial delay..." wording here could still use some updating.